CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
 
 ...
 
+## 1.4.13 (15.07.2020)
+
+### Enhancements
+
+####([1232](https://github.com/allegro/hermes/pull/1232)) Serialization with schema id
+
+Issues resolved: [1225 - Confluent message serialization and deserialization compataibility](https://github.com/allegro/hermes/issues/1225)
+and [682 - Avro messages does not contain header](https://github.com/allegro/hermes/issues/682).
+
+####([1235](https://github.com/allegro/hermes/pull/1235)) Docker update
+
+Dockerfiles have been updated. The project is now automatically built by dockerhub.
```

Comment:
Probably it's not build by dockerhub, but is published at dockerhub, am I wrong?

Reply:
Dockerhub has now automated builds and this is the feature we are using now :) More info [here](https://docs.docker.com/docker-hub/builds/#:~:text=Docker%20Hub%20can%20automatically%20build,to%20build%20into%20Docker%20images.)

Reply:
indeed, sorry :(